lib/sisimai/bite/email/courier.rb

```diff
@@ -0,0 +1,265 @@
+module Sisimai::Bite::Email
+  module Courier
+    # Sisimai::Bite::Email::Courier parses a bounce email which created by Courier
+    # MTA. Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Courier.pm
+      require 'sisimai/bite/email'
+
+      # http://www.courier-mta.org/courierdsn.html
+      # courier/module.dsn/dsn*.txt
+      Re0 = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->